scripts/mkstatic.py

```diff
@@ -265,7 +277,7 @@ def write(self):
         # which comes in as a scalar for any potential block/thread via the argument list.
         ccpp_var = None
         parent_standard_names = []
-        for ccpp_stage in CCPP_STAGES:
+        for ccpp_stage in CCPP_STAGES.keys():
```

Comment:
Isn't the `.keys()` here redundant? Same question on line 303 below.

Reply:
Not sure, haven't tried. I generally prefer to add `.keys()`, because it makes it explicit what we are looping over.